packages/aws-cdk-lib/aws-apigatewayv2-integrations/README.md

```diff
@@ -210,3 +211,43 @@ webSocketApi.addRoute('sendMessage', {
   integration: new WebSocketLambdaIntegration('SendMessageIntegration', messageHandler),
 });
 ```
+
+### AWS WebSocket Integration
+
+AWS integrations enable integrating with an AWS service action, including the Lambda function-invoking action. With the Lambda function-invoking action, this is referred to as the Lambda custom integration. With any other AWS service action, this is known as AWS integration. Supported only for WebSocket APIs.
```

Comment:
This is a direct quote from [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-integration.html#cfn-apigatewayv2-integration-integrationtype). Generally, if we want to link to documentation from elsewhere thats fine. But I think we should express what the CDK construct is doing in our own words.